IPython/core/magics/script.py

```diff
@@ -8,9 +8,15 @@
 import sys
 import signal
 import time
-from subprocess import Popen, PIPE, CalledProcessError
+import asyncio
 import atexit
 
+try:
+    from asyncio import create_task
+except ImportError:
+    # 3.6 had this function in a different module, this can go away when 3.6 is dropped
+    from AbstractEventLoop import create_task
```

Comment:
3.6 isn't supported beyond IPython 7.16 so I guess I can drop all of this? I'll have test failures in travis, though...